tests/crypto_test.py

```diff
@@ -66,29 +64,54 @@ def tearDownModule():
             os.kill(int(pid), signal.SIGKILL)
 
 
+def make_key(revoked=False, expired=False, invalid=False, can_encrypt=True,
+             can_sign=True):
+    # This is ugly
+    mock_key = mock.create_autospec(gpg._gpgme._gpgme_key)
```

Comment:
That is correct and part of the "new" bindings. Try `python2 -c 'import gpg; print gpg._gpgme; print gpg._gpgme._gpgme_key'` in the shell.